_targets.R

```diff
@@ -84,16 +84,13 @@ dummy_date <- "2022-06-15"
 #2) Configure model inputs/variables 
 
 # Define test and validation sites
-val_sites <- c("01472104", "01473500", "01481500")
-tst_sites <- c("01475530", "01475548")
+val_sites <- c("01475530", "01475548")
 
 # Define train/val/test dates
 train_start_date <- '1980-01-01'
-train_end_date <- '2014-10-01'
-val_start_date <- '2014-10-01'
-val_end_date <- '2015-10-01'
-test_start_date <- '2015-10-01'
-test_end_date <- '2022-10-01'
+train_end_date <- '2015-10-01'
+val_start_date <- '2015-10-01'
+val_end_date <- '2021-10-01'
```

Comment:
@jsadler2 did you mean to stop the validation time period in 2021? The previous `test_end_date` went out to `'2022-10-01'`. I'm also asking because I'm wondering if I should extend the NWIS pull date beyond the `latest_date` which is currently set to `"2021-12-31"`.  

```suggestion
val_end_date <- '2022-10-01'
```

Reply:
See #135 for the resolution of this